v3/boardsource/the_q/the_q.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "The Q",
+  "vendorId": "0x4273",
+  "productId": "0x7552",
+  "matrix": {"rows": 7, "cols": 6},
+  "keycodes": ["qmk_lighting"],
+  "menus": ["qmk_rgb_matrix"],
```

Comment:
This enables all animations, just making sure thats what you want? QMK source only enables a handful it looks like, would require a custom menu to show the correct amount

Reply:
Sorry this was forgotten, fixed now.

Reply:
To have the correct options shown, you should add the custom menu RGB Matrix explicit declaration and only use the effects you have enabled by removing the unused ones from the option list: https://www.caniusevia.com/docs/built_in_menus#qmk_rgb_matrix

Reply:
Understood, thanks for the pointer.